src/chapters/strings/string-module.rst

```diff
@@ -127,20 +127,4 @@ Check Your Understanding
 
 .. Answers = a, c, d, g
 
-.. raw:: html
 
-   <script type="text/JavaScript">
-      function highlight(id, answer) {
-         text = document.getElementById(id).innerHTML
-         if (text.indexOf('Correct') !== -1 || text.indexOf('Nope') !== -1) {
-            return
-         }
-         if (answer) {
-            document.getElementById(id).style.background = 'lightgreen';
-            document.getElementById(id).innerHTML = text + ' - Correct!';
-         } else {
-            document.getElementById(id).innerHTML = text + ' - Nope! This is True.';
-            document.getElementById(id).style.color = 'red';
-         }
-      }
-   </script>
```

Comment:
Reading this question again, I think that the `highlight` function might not be the best choice.

For options a-g, let's try calling the `revealTrueFalse` function instead of `highlight`.  We should also rephrase the question to better fit the new answer function.

Reply:
check this one, Not sure its right

Reply:
It's working fine.  I just added a space after each option to separate them from the revealed answer.